docs/Using-Fleet/MDM-commands.md

```diff
@@ -71,5 +71,18 @@ Fleet UI:
 
 2. Look at the on-screen information.
 
+## List of recent commands
```

Comment:
```suggestion
## List recent commands
```

Minor nit. I think we can make "list" a verb here. This communicates to the user that they should follow this guide if they want to list recent commands.